Documentation/docs.md

```diff
@@ -37,6 +37,7 @@ Administrators who need to create reliable and scalable key-value stores for the
  - [Performance][performance]
  - [Versioning][versioning]
  - [Supported platform][supported_platform]
+ - [Authentication][authentication]
```

Comment:
put between security and monitoring? seems like it belongs closer to the setup stuff